docs/AppService/PerformAppServiceInteraction/assets/PerformAppServiceInteractionFromHMI.puml

```diff
@@ -0,0 +1,14 @@
+@startuml
+
+participant HMI
```

Comment:
I disagree, mainly just because this repository is meant to describe the HMI's role in SDL. I think for the sake of consistency that just keeping the normal terminology of `HMI` is better.